cypress/integration/metaNewsSpec.js

```diff
@@ -41,7 +41,7 @@ describe('Article Meta Tests', () => {
     cy.get('script')
       .last()
       .should('have.attr', 'src')
-      .and('match', /(\/static\/js\/client-\w+\.\w+\.js)/g);
+      .and('match', /(\/static\/js\/main-\w+\.\w+\.js)/g);
```

Comment:
```suggestion
      .and('match', /(\/static\/js\/\w+-\w+\.\w+\.js)/g);
```
Suggested change, so that if we change `main` to something else this test still works.

Reply:
Not a bad suggestion but I like being alerted when the name of our outputted files has changed. Suggest we stick as-is.